.travis.yml

```diff
@@ -35,8 +43,16 @@ aliases:
   - &run-integration-tests |
     set -e
     jdk="$(ext/travisci/jdk-from-spec "$PDB_TEST")"
-    if test "$(uname)" = Linux; then jdk_switcher use "$jdk"; fi
     ext/travisci/prep-os-essentials-for "$PDB_TEST"
+    if test "$(uname)" = Linux; then
+      jdk_switcher use "$jdk"
+    elif test "$(uname)" = Darwin; then
+      JAVA_HOME=$(/usr/libexec/java_home -v $1);
```

Comment:
I think it'd be worth elaborating on what this is/does in the commit message.  And is it just finding some built-in java?  If so, then do we still need the cask?

Reply:
This is specific to OSX but it will find any installed JDK version whether it's Oracle or OpenJDK. I'll add a comment.